.github/workflows/sanity.yml

```diff
@@ -90,11 +91,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        KUBERNETES_VERSIONS: ["1.20.2"]
+        kubernetes_version: ["1.20.7"]
+        include:
+          - kubernetes_version: "1.20.7"
+            kind_image: "1.20.7@sha256:cbeaf907fc78ac97ce7b625e4bf0de16e3ea725daf6b04f930bd14c67c671ff9"
+            kind_version: "v0.11.1"
     env:
-      KUBERNETES_VERSION: ${{ matrix.KUBERNETES_VERSIONS }}
-      KIND_VERSION: "v0.10.0"
```

Comment:
v0.10.0 started failing in the last 24h due to cni network not being ready. Moving to v0.11.1 fixes this issue.